hibernate-core/src/main/java/org/hibernate/persister/entity/EntityPersister.java

```diff
@@ -811,8 +811,4 @@ Object createProxy(Serializable id, SharedSessionContractImplementor session)
 	int[] resolveAttributeIndexes(String[] attributeNames);
 
 	boolean canUseReferenceCacheEntries();
-
-	default boolean canIdentityInsertBeDelayed() {
-		return false;
-	}
```

Comment:
Removing this will break applications that implement this method. You should probably add it back in and deprecate it and say it's no longer used.

Reply:
Added back as a deprecated method in the last PR resync.